app/components/InputLabel.tsx

```diff
@@ -1,9 +1,24 @@
 import { PropsWithChildren } from "react";
+import { tv } from "tailwind-variants";
 
-export default function InputLabel({ children }: PropsWithChildren) {
-  return (
-    <label className="font-manrope text-primary-600 font-normal text-xs tracking-wide">
-      {children}
-    </label>
-  );
+type InputLabelProps = {
+  inputState: "error" | "filled" | "empty";
```

Comment:
suggestion: `inputState` -> `state` 
The component name (InputLabel) already implies that the state relates to an input so this change will make the code more concise while still being fully descriptive.